### ISSUE TITLE
Add detaching enchantment

### ISSUE THREAD

**Issue by Touch-Night** [Ladysnake/Requiem#535]
It's a feature request.
Detaching enchantment can be applyed to swords.
You can get detaching enchanted book on a anvil(or maybe through mortician?) with humanity enchanted book and a soul vessel. After that, the soul vessel will broke.
Swords with detaching enchantment give only mortal a fixed possibility to force the soul out of the body for a while. Which means that the target will lose its AI when its soul is out, and get it back as the soul automatically flys back into the body. And if a body is currently possessed by a remnant, it will forcibly spilt it out of the body. No effect when remnant is using it.
Higher the level, larger the chance.

**Comment by Touch-Night** [comment]
or maybe the enchantment can also be called Banishment?

**Comment by Pyrofab** [comment]
Interesting idea, not sure about adding it to the base mod though. Have you tried using a Penance obelisk to force a remnant out of a body ?

**Comment by Touch-Night** [comment]
yeah but the Penance obelisk only works for remnant, what i am thinking about is that the sluggish state when the soul is temporary out of an entity may help mortal beat it. And the effect on the creature possessed by remnant is a icing on the cake.

**Comment by Asteroid4** [comment]
This seems like a great idea.
